bigtable/doc/bigtable-hello-world.md

```diff
@@ -0,0 +1,90 @@
+# Example: C++ Hello World Application
+
+This example is a very simple "hello world" application, written in C++, that
+illustrates how to:
+
+* Connect to a Cloud Bigtable instance.
+* Create a new table.
+* Write data to the table.
+* Read the data back.
+* Delete the table.
+
+## Running the example
+
+This example uses the
+[Cloud Bigtable C++ Client Library](https://GoogleCloudPlatform.github.io/) to
```

Comment:
This URL seems wrong... it redirects to https://github.com/GoogleCloudPlatform which is probably not what you want. Did you mean https://googlecloudplatform.github.io/google-cloud-cpp/ ?

Reply:
Yup.  Fixed.